packages/circuit-ui/components/Selector/Selector.tsx

```diff
@@ -77,7 +77,9 @@ export interface SelectorProps
 type LabelElProps = Pick<SelectorProps, 'disabled' | 'size' | 'noMargin'>;
 
 const baseStyles = ({ theme }: StyleProps) => css`
-  display: inline-block;
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
```

Comment:
Not directly related to this PR, but does the border properties set between lines 87-88 serves a purpose, since it's actually being set in the `&:before` selector?

Reply:
`border: none;` is used to remove any default border, though the `label` element doesn't have a border by default. It could probably be removed.

`border-radius` is used to round the corners. It's needed on both the element and the `before` pseudo-element to ensure that the content doesn't overflow in the corners.

Reply:
Thanks!